guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -115,13 +115,6 @@
     background-image: url("images/guac-tricolor.png");
 }
 
-.login-ui.continuation div.login-dialog {
-    border-right: none;
-    border-left: none;
-    box-shadow: none;
-    max-width: 6in;
-}
-
```

Comment:
So, I removed all of this because I wasn't sure that the continuation piece of this actually does anything other than make the dialog box look a little goofy.  I can understand hiding the logo and version, which I did leave in place; however, this just takes off half of the borders, the shadow, and makes the box show up as an odd width (relative to the "normal" login box).  I also noticed this when working on the RADIUS module - when it prompts for the additional credentials from RADIUS.

Any reason not to pull this out?

Reply:
Added back the max-width tag so that the message isn't quite so squished, but kept the shadow/box around it.